### PR TITLE
Plans grid: monthly plans copy changes 

### LINKED ISSUE #49390: Plans grid: copy changes after Monthly plans
## What

While working on #48963, we had to re-use some existing text strings because the ones requested in the designs were not already translated

## Why

We should open a new PR with the exact copy requested in the design, and merge it once the translations are available

## A/C

- [x] PR should have the `String freeze` label, and be merged only when all translations are available
- [x] Toggle component: From  "Pay monthly" / "Pay annually"  to  "Monthly" / "Annually"
- [x] Plan item / card (_accordion_ variant)
  - [x] From "billed annually" to "billed as %s annually" (since the price already has "/month")
  - [x] From "per month, billed monthly" to "billed monthly" (since the price already has "/month")
- [x] Plan item / card (_table_ variant): From "billed annually" to "per month, billed as %s annually"
- [x] Final checkout step of Step by Step flow: From "billed annually" to "per month, billed as %s annually"

### PR DESCRIPTION
**Pull request by ciampo**
#### Changes proposed in this Pull Request

* Change copy related to the plans grid

Do not merge this PR until all translations have been completed!

### Setup
- prepare your sandbox
- in two separate terminal windows, run:
  - `yarn start` 
  - `cd apps/editing-toolkit && yarn dev --sync`

### Focused Launch

1. Assign yourself to the "treatment" group in the `focused_launch_flow_v2` A/B test (id `20106`)
2. Add an _unlaunched_ site created through `/start`to your sandbox
3. Open the block editor (e.g. by visiting `calyspo.localhost:3000/page/UNLAUNCHED_SITE_CREATED_WITH_START.wordpress.com/home`)`
4. Press "Launch" button, Focused Launch summary view should appear
5. Click on "View all plans" link
   - [x] Verify that the new copy is correct in the plans details view (table variant), according to A/C in #49390
   - [x] These copy changes are not expected to affect non-english locales for the purpose of this PR (but it's important that the previous _translated_ fallback text is used)

### Step by Step Launch

1. Add an _unlaunched_ site on a _free_ plan created through `/new` to your sandbox
2. Visit `wordpress.com/page/SITE_ID.wordpress.com/home`
3. Press "Launch" button, Step by Step Launch modal should appear.
4. Navigate to the plans step
   - [x] Verify that the new copy is correct according to A/C in #49390 (accordion variant)
   - [x] These copy changes are not expected to affect non-english locales for the purpose of this PR (but it's important that the previous _translated_ fallback text is used)

5. Navigate to the final (checkout) step
   - [x] Verify that the new copy is correct according to A/C in #49390
   - [x] These copy changes are not expected to affect non-english locales for the purpose of this PR (but it's important that the previous _translated_ fallback text is used)

### Gutenboarding

1. Visit `/new` and navigate to the plans step
   - [x] Verify that the new copy is correct in the plans details view (accordion variant), according to A/C in #49390

#### Screenshots

For translators: 

image

image

image

image

image

Fixes #49390
Relates to #47576

### CODE REVIEW

File: apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx
Comment (a8ci18n) [inline apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'per month{{br/}}billed yearly' )` *ES Score: 7*

File: packages/plans-grid/src/plans-table/plan-item.tsx
Comment (a8ci18n) [inline packages/plans-grid/src/plans-table/plan-item.tsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **28** times:
`translate( 'Billed monthly' )` *ES Score: 14*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feat/plans-grid-copy-changes)

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5477733

Thank you @ciampo for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5477733).

File: packages/plans-grid/src/plans-table/plan-item.tsx
Comment (ciampo) [inline packages/plans-grid/src/plans-table/plan-item.tsx:233]: Removed unnecessary `sprintf` call, since `fullWidthCtaLabelSelected` doesn't have any placeholder text

File: apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx
Comment (StefanNieuwenhuis) [inline apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx]: I know that there's a lot of discussion going on about `i8n` and how to implement methods properly, so it's no biggie if it's not possible but can't we "merge" `planSummaryCostLabelAnnually` & `planSummarCostLabelMonthly` into 1 `const` by creating a ternary based on `planProduct.billingPeriod`?

File: packages/plans-grid/src/plans-accordion-item/index.tsx
Comment (StefanNieuwenhuis) [inline packages/plans-grid/src/plans-accordion-item/index.tsx]: Same here (see my previous comment).

File: packages/plans-grid/src/plans-table/plan-item.tsx
Comment (StefanNieuwenhuis) [inline packages/plans-grid/src/plans-table/plan-item.tsx]: Same here (see comment above)

**Review by StefanNieuwenhuis — CHANGES_REQUESTED** [review]
#### Focused Launch - Plans Grid (table variant)
The screenshots for translators don't align with the AC in #49390 and the translations in the code. Can you please update them?

Screenshot 2021-02-08 at 10 49 03

Screenshot 2021-02-08 at 10 46 36

File: apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx
Comment (ciampo) [reply apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx]: I'm afraid we can't — the whole point of keeping these variables separate from the ternary operator is to somehow avoid an optimisation from terser that breaks `i18n` (see p1612522813165500/1612255293.049500-slack-C013QHLF28Y)

File: packages/plans-grid/src/plans-accordion-item/index.tsx
Comment (ciampo) [reply packages/plans-grid/src/plans-accordion-item/index.tsx]: Same as https://github.com/Automattic/wp-calypso/pull/49397/files/2d137c3873faedb724b186c106063d507fabed6c#r571922139

File: packages/plans-grid/src/plans-table/plan-item.tsx
Comment (ciampo) [reply packages/plans-grid/src/plans-table/plan-item.tsx]: Same as https://github.com/Automattic/wp-calypso/pull/49397/files/2d137c3873faedb724b186c106063d507fabed6c#r571922139

**Comment by ciampo** [comment]
> The screenshots for translators don't align with the AC in #49390 and the translations in the code. Can you please update them?

@StefanNieuwenhuis I've updated the screenshots, and I've [replied](https://github.com/Automattic/wp-calypso/pull/49397#discussion_r571922139) to your comment —  can you please take another look?

File: apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx
Comment (StefanNieuwenhuis) [reply apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx]: Thanks for elaborating. Makes sense and I hope we find a solution in the future.

**Review by StefanNieuwenhuis — APPROVED** [review]

**Comment by ciampo** [comment]
On hold until all strings are translated

**Comment by ciampo** [comment]
@Automattic/i18n There seems to be an issue with the translations on this PR — Everything should already be translated, but I did one last check before merging and found out that the copy is still appearing in english (the same issue was flagged on a different PR, https://github.com/Automattic/wp-calypso/pull/49290#issuecomment-787778243)

Can you help debugging what's the issue?

**Comment by ciampo** [comment]
@StefanNieuwenhuis, I pushed some changes so that the UI will display the old text until the new text is fully translated. This is necessary for ETK related changes.

I updated the testing instructions reflecting the latest changes

**Review by yansern — APPROVED** [review]
Tested on Focused Launch & Step By Step Launch. LGTM.

**Pull request merged**
